x/sla/keeper/price_feeds_test.go

```diff
@@ -344,6 +344,8 @@ func (s *KeeperTestSuite) TestRemovePriceFeeds() {
 }
 
 func checkEquality(t *testing.T, sla1, sla2 slatypes.PriceFeed) {
+	t.Helper()
```

Comment:
wut do this?

Reply:
this just designate that it is a helper (visually) and this also changes how this function is logged when running verbose tests